src/renderer/components/+catalog/catalog-category-label.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import React from "react";
+import type { CatalogCategory } from "../../../common/catalog/catalog-entity";
+
+interface CatalogCategoryLabelProps {
+  category: CatalogCategory;
+}
+
+/**
+ * Display label for Catalog Category for the Catalog menu
+ */
+export const CatalogCategoryLabel = ({ category }: CatalogCategoryLabelProps) =>
+  (
+    <div className="flex">
+      <div>{category.metadata.name}</div>
+      <div className="flex items-center">{category.getBadge()}</div>
```

Comment:
I think it's better not to render container div if there's no badge.

Reply:
Good points, changed